cloud-providers/aws/beanstalk/aws-eb-docker-remote-deployment/README.md

```diff
@@ -0,0 +1,36 @@
+# Deploy an application on AWS Elastic Beanstalk using a Docker image from a Docker repository
+
+AWS Elastic Beanstalk (EB) allows deploying applications using an existing Docker image in a Docker repository. EB will need a `Dockerrun.aws.json` configuration file to deploy and run the application in one or more Docker containers. This JSON file specifies the name of the Docker image to use and overrides the default configuration of that image. EB will automatically scale and create/drop Docker container as needed. EB also automatically creates a load-balancer in front of the containers.
```

Comment:
Typo

>EB will automatically scale and create/drop Docker container as needed.

EB will automatically scale and create/drop Docker container**s** as needed.